Sources/Widgets/Widgets3D/AngleWidget/behavior.js

```diff
@@ -92,7 +92,9 @@ export default function widgetBehavior(publicAPI, model) {
 
       if (
         worldCoords.length &&
-        (model.activeState === model.widgetState.getMoveHandle() || isDragging)
+        (model.activeState === model.widgetState.getMoveHandle() ||
+          isDragging) &&
+        model.activeState.setOrigin
```

Comment:
You might want to explain when setOrigin does not exist, e.g. `model.activeState.setOrigin // e.g. the line is pickable/activable but not draggable`